README.md

```diff
@@ -97,120 +97,112 @@ Alternatively, follow these steps to install the plug-in manually:
-  "device-id"     : "my-selected-device-id",
+  "product-id"    : "my-model-id",
+  "devicegroup-id": "my-selected-device-id",
+  "deployment-id" : "deployment-new" or "my-deployment-id"
   "device-file"   : "src/device.nut",
-  "agent-file"    : "src/agent.nut",
+  "agent-file"    : "src/agent.nut"
 }
 ```
 
-When a project is created, empty device and agent code files (`device.nut` and `agent.nut`) are automatically created 
+When a project is created, empty device and agent code files (`device.nut` and `agent.nut`) are automatically created
 and stored in the `<Project Name>/src` folder.
 
-If a project is created successfully, a new window with the project folder is opened. 
+If a project is created successfully, a new window with the project folder is opened.
 
 **NOTE**: If you need to apply the Squirrel language syntax highlighting to files with extension other than `.nut`
-please make sure you have Squirrel (Electric Imp) language selected under the `View` > `Syntax` menu item. 
+please make sure you have Squirrel (Electric Imp) language selected under the `View` > `Syntax` menu item.
 
 ### Opening an Existing Project
 
-To open an existing Electric Imp project, select the `Project` > `Open Project...` menu option and choose the 
+To open an existing Electric Imp project, select the `Project` > `Open Project...` menu option and choose the
 `<Project Name>.sublime-project` file from your project folder.
 
 **NOTE: The plug-in won't properly detect Electric Imp project if it is not opened as described, ie. if it is opened
 as a folder, not as a Text Sublime project!**
 
 ### Building and Running
 
-To build and deploy the application code, please select the `Tools` > `Packages` > `Electric Imp` > `Build and Run` 
-menu item. This action uploads the agent and the device code to the server, 
+To build and deploy the application code, please select the `Tools` > `Packages` > `Electric Imp` > `Build and Run`
+menu item. This action uploads the agent and the device code to the server,
 and restarts all of the devices assigned to the model.
 
-When you build code (or perform any other action that requires access to the impCloud&trade;) 
+When you build code (or perform any other action that requires access to the impCloud&trade;)
 for the first time, you will be asked to provide:
 
 - The path to the Node.js executable (if it is not automatically detected by the plug-in).
 - The location of the Builder *cli.js* command line tool (if not automatically detected by the plug-in).
-- Your Build API Key, which can be obtained at the [Electric Imp IDE](https://ide.electricimp.com/) by clicking on the 
-`username` > `Build API Keys` menu item at the top right of the screen. 
-- A model name. A model with this name will be created for the project, 
-or if the name matches an existing model, that will be used instead.
+- User name and password, which you use for the [Electric Imp IDE](https://impcentral.electricimp.com/)
```

Comment:
`[Electric Imp impCentral](https://impcentral.electricimp.com/)`

Reply:
Replaced IDE -> impCentral, checked that reference point on impCentral